js/apps/admin-ui/src/realm-settings/EmailTab.tsx

```diff
@@ -66,21 +68,6 @@ export const RealmSettingsEmailTab = ({
     defaultValue: "",
   });
 
-  const save = async (form: RealmRepresentation) => {
-    try {
-      const savedRealm = { ...realm, ...form };
-
-      // For default value, back end is expecting null instead of empty string
-      if (savedRealm.smtpServer?.port === "") savedRealm.smtpServer.port = null;
-
-      await adminClient.realms.update({ realm: realmName }, savedRealm);
-      setRealm(savedRealm);
```

Comment:
Looks like this is the only place `setRealm` was called, so it's no longer needed to keep it in a separate state. CAn you remove the `setState()` call that this originates from?